src/main/java/com/softteam/vocabuilder/persistence/entity/Category.java

```diff
@@ -0,0 +1,23 @@
+package com.softteam.vocabuilder.persistence.entity;
+
+
+import lombok.Data;
+
+import javax.persistence.*;
+import java.util.Date;
+@Data
+@Entity
+@Table(name = "category")
+public class Category {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
`AUTO` incremental is not very used nowadays, since it's very evident how many elements do you have
let's use in the project UUID (If you don't know what it is, read a bit, it's part of the software development process, don't worry)

Reply:
I did a little research and as a result add the dependency to the pom.xml
        <dependency>
            <groupId>com.fasterxml.uuid</groupId>
            <artifactId>java-uuid-generator</artifactId>
            <version>4.0.1</version>
        </dependency>
Also change all Lont type data a UUID.